### DISCUSSION TITLE
How to revert E2 Synth version to the original firmware

### DISCUSSION THREAD

**Discussion by AndrewSosna** [bangcorrupt/hacktribe#70] [Q&A] [answered]
Hi. First of all thank you for this hack, I've successfully applied it to my e2synth using macOS, but just to be safe and to compare with original PCM samples I'd like to revert the hack temporary.
I understand that I should use the original 2.02 firmware for synth, but the questions are:
1. What folder path should I use? it's a blue synth version with already hacked firmware. I assume it should be /sampler
2. Should I do any modification of header of the original firmware? I guess it won't accept it without a modified header because, probably, now it thinks that it's a sampler so it would want a sampler firmware, right?

Thanks in advance and sorry for the stupid questions.

**Comment by bangcorrupt** [comment, marked as answer]
These are sensible questions and I'll try to document them in the wiki soon. You are correct on both counts.  Hacktribe thinks it is a sampler, the header will need modifying to look like the sampler firmware update file.  Run `e2-header.py` with the `sampler` option to set the header as a sampler file.

    python scripts/e2-header.py SYSTEM.VSB sampler

Since 3d31b7c,  Hacktribe will look for files under the `hacktribe` directory.

Update:

See wiki section [Revert to Factory Firmware](https://github.com/bangcorrupt/hacktribe/wiki/How-To#revert-to-factory-firmware).

**Reply by frndob** [reply]
Hi! please explain how to revert..I edit header, and use de GUI for patching..and the hacktribe firmware still update.

**Reply by bangcorrupt** [reply]
Don't patch the firmware if reverting, just edit the header of the factory firmware and put it at `KORG/hacktribe/System/SYSTEM.VSB`.  The GUI doesn't support this yet, follow the comment above.

**Reply by frndob** [reply]
Thanks..but how to edit the header? the factory firmware for electribe 2 or electribe sampler? I read your answer, but I don't understand how to run the e2-header.py without the GUI

**Reply by bangcorrupt** [reply]
python scripts/e2-header.py SYSTEM.VSB sampler

**Reply by frndob** [reply]
Sorry..I don't understand..this line code in python?? I use mac, in python shell, says "sintax error". Be kind please, I'm totally noob.

**Reply by bangcorrupt** [reply]
Run it in the terminal, not the python interpreter.

**Reply by Ondrekk12** [reply]
> scripts

3d31b7c92ba5616ffd67d1dc6c8739816813da00

**Reply by Ondrekk12** [reply]
>     python scripts/e2-header.py SYSTEM.VSB sampler

python scripts/e2-header.py SYSTEM.VSB sampler

**Comment by AndrewSosna** [comment]
Awesome, thank you, I've reverted my Synth back, no issues. 

Would be nice to have some way to extract PCM samples from synth version somehow, as a starting point for Hacktribe, but that's  a different story, whoever switches from synth to sampler is already bored with these default samples.

**Reply by bangcorrupt** [reply]
I can make this work at some point.  When the PCM data is loaded from the flash, it's decompressed and sent to DSP RAM one word at a time.  I can hijack this to copy PCM to CPU RAM, then dump it over sysex as a single blob.  If I can get a debugger attached to the DSP then I may be able to fix the issue with sample saving and have the factory samples loading like normal.

**Reply by ShoveD01** [reply]
Would I be right in saying, in order to update from the previous hacktribe version to the latest hacktribe version , no modification of the header is required. Because hacktribe thinks it's a sampler, and is being updated by a patched sampler firmware.

**Reply by bangcorrupt** [reply]
Correct.  Modifying the header is only necessary if you currently have factory synth firmware installed, or are reverting from Hacktribe to synth firmware.

**Reply by AndrewSosna** [reply]
Just curious, I see that the first difference in headers is just a letter "s" instead of just white space, but what's the second one? Just some character for CRC validation?

**Reply by bangcorrupt** [reply]
Device ID numbers are 123 for synth and 124 for sampler.  They're used in sysex messages and a couple of places in the firmware and file headers.  

Thank you so much for your support! :smiley:

**Reply by AndrewSosna** [reply]
Thank you for doing what Korg should have done in the first place!

**Comment by histmaster** [comment]
I tried to turn my gray hacktribe back to the original firmware. Always get an invalid file error. 
What I did:
1) downloaded the firmware from korg
2) patched the header with the es2-header script
3) copied the patched file to hacktribe/system/

any suggestions?

**Reply by bangcorrupt** [reply]
Can you email me the file you're trying to install?

**Reply by histmaster** [reply]
On the way

**Reply by bangcorrupt** [reply]
You need to edit the header to make it look like the sampler firmware.  Call [e2-header.py](https://github.com/bangcorrupt/hacktribe/blob/main/scripts/e2-header.py) with the **sampler** argument:

    python e2-header.py SYSTEM.VSB sampler

**Reply by histmaster** [reply]
Thank you worked!

**Reply by herrmautsch** [reply]
Hey @histmaster could you provide the File I’m trying to revert too and honestly I have no idea what all of this header changing python terminal words mean ;-)

you’d be an awesome help

@bangcorrupt i seem to have issues with the midi connection neither the usb midi nor the midi in/out seem to work properly… is this a know issue?

thanks a lot for the hacktribe work though it’s really fun to play around with.

**Reply by bangcorrupt** [reply]
MIDI works for me, I don't know of any issues.  Does MIDI work for you using the factory firmware?

**Reply by herrmautsch** [reply]
I will have to try, therefore I need to switch back. Being a total noob that’s kind of a hustle for me ;-)

**Reply by bangcorrupt** [reply]
See wiki section [Revert to Factory Firmware](https://github.com/bangcorrupt/hacktribe/wiki/How-To#revert-to-factory-firmware).

**Reply by luciferseamus** [reply]
@herrmautsch 
Did you find the file you were looking for?
I too am in the same boat.  I would like to restore my E2 to factory settings and the 202 firmware just gives me an error.  I am fairly computer literate but this is beyond my scope of knowledge.

@bangcorrupt 
I super appreciate your work on the project and since installing hacktribe I have found zero errors (I am a simple user).  I simply have found that I really have no need for a sampler in my work flow and feel I can get more use out of the original firmware. Me and ol' grey work well together and I feel like adding blue to the mix will be a good thing.  who knows. . . 
anyway, I take it that it is "easy" to do just modify the header of the .VSB file by way of. . . .  magic? I don't know how to edit this file and although I downloaded the GUI I am getting and error regarding some .dll file. like I said I really don't know what I am doing with all this.  is there any way you can provide the file needed to go back to the factory/202 firmware settings?
Thank you very much again for all of your hard work.  you pulled off something I don't think I will ever understand and did it triumphantly!

**Reply by bangcorrupt** [reply]
This isn't a feature in the GUI yet, you need to run the python script in a terminal.  See the instructions in the wiki linked above.

**Reply by luciferseamus** [reply]
I did have a look at the wiki prior to posting and I am guessing terminal is a linux feature? is that correct? I am running windows 7 and the closest thing I have to that is command prompt, all I get back from running the commands there is an error. I am guessing I need to download additional programs in order to do this on a windows box? if so what? Is there anyway you can provide the edited file needed to go back to the stock firmware?

THanks a million in advance for any and all help you can provide!

**Reply by tokejuhoklee** [reply]
What is the error you're getting? If executing a python script, doing it from a command prompt in windows shouldn't be different than doing it from a terminal. Guessing without wanting to make assumptions you may not have installed python on your system to be able to execute it from the command prompt?

**Reply by luciferseamus** [reply]
> What is the error you're getting? If executing a python script, doing it from a command prompt in windows shouldn't be different than doing it from a terminal. Guessing without wanting to make assumptions you may not have installed python on your system to be able to execute it from the command prompt?

This is likely correct. I do not have python on my system as I have no real need for it. . .  Other than to edit this one file that is. That's kinda why I was hoping someone could make available the file needing to be edited. If I must do it myself is there a lite version I can use to accomplish this task? is it a single download or do I need to incorporate a few. . . repositories/resources/supplemental elements? sorry I really don't know much about this.

Thank you for taking the time to respond and help me. I really appreciate your time and wisdom.

**Reply by bangcorrupt** [reply]
A standard python installation is all you need, there are lots of beginner tutorials online.  The commands in the wiki will work if the script and firmware file are in the same directory.

**Reply by luciferseamus** [reply]
@bangcorrupt @tokejuhoklee @herrmautsch @AndrewSosna @didak00 
> A standard python installation is all you need, there are lots of beginner tutorials online. The commands in the wiki will work if the script and firmware file are in the same directory.

Thank you for responding. As it turns out I keep running into road blocks when trying to install python.  Missing Dll files, I am not running the most current version of windows so some packages do not have installers blah blah blah. . . .

I could try and chase down and correct all these issues but in the end I only need one file edited. It really doesn't make sebse to install a bunch of stuff on my system all so I can alter this one file.  So, I ask again. Is there any one out there who could make available the altered OEM file so I can revert back? 
I would really appreciate it! 
Since it seems as though others in this thread have accomplished the editing of the file it would be awesome if someone could take one more step and upload it and help me and other fellow music makers out. Please and thank you.

@histmaster 
> Thank you worked!

is there anyway you can send me the file you got working so I can get my tribe back to factory?

**Reply by bangcorrupt** [reply]
I won't distribute copyrighted files from this repo. There needs to be a line somewhere and this seems a reasonable place to put it.  I agree that it would be useful to have an archive of patched firmware files somewhere on the internet.  

You can use a bash shell with python installed in [github codespaces](https://github.com/codespaces).  

1. Select the 'Blank' template and wait for it to load.  The lower half of the horizontal split is a bash shell.
2. Right click in the vertical pane on the left, select `Upload` and upload synth firmware `SYSTEM.VSB` from your local filesystem.
3. In the bash shell, clone the Hacktribe repo: 
 
        git clone https://github.com/bangcorrupt/hacktribe.git

5. Run the script to edit the header, making synth firmware look like sampler firmware:

        python hacktribe/scripts/e2-header.py SYSTEM.VSB sampler

6. The script will overwrite the original file.  Right click `SYSTEM.VSB` in the vertical pane on the left and select `Download`.

**Reply by luciferseamus** [reply]
>I won't distribute copyrighted files from this repo. There needs to be a line somewhere and this seems a reasonable place to put it.

@bangcorrupt : I completely understand not wanting to host and thereby infringe on copyrighted material.  that is reasonable and sensible.  


With that, I have to say thanks for pointing out "github codespaces". . . .  I was unaware that was even a thing as this sort of stuff is not my milieu.  I was able to follow your instructions to the T and everything worked out perfectly!
Thank you very much.

**Comment by didak00** [comment]
Hi. Anybody can help me? Please..i m just two days fighting without result. More simple
I buy a electribe 2 synth grey.
Update to 2.02 korg firmware
Install the hacktribe firmware. The problems begin
I m watch in the boot the logo but dont load all the sounds and more fast to load..i think that not boot correctly, after i m only have the synth sounds. Not samples, no kicks...nothing.
I m try to return to old version following the manual.
1. Download the 2.0 electribe sampler oficial
2. Install git and i apllied the patch at firmware. It seems good. With the patched tool
3. After copy and rename de SYSTEM file in all the diectories combinations...Hacktribe/system...korg/system...sample/system...nothing to do. No file...invalid file...different errors...
I m just only wish come back to the original firmware electribe2 grey synth...somebody can help me please? I m tried a lot of time
Thanks

**Reply by didak00** [reply]
Now im try to format the card and create the folder structure as a sampler electribe..not as the synth...im turned crazy...i have a electribe synth sampler without sounds 😒

**Reply by bangcorrupt** [reply]
Hacktribe will look for files under the `KORG/hacktribe/` directory, paths are case sensitive.

Don't patch the firmware if reverting to the factory firmware, just edit the header.  To revert to the factory firmware put the factory firmware with edited header at `KORG/hacktribe/System/SYSTEM.VSB`.

> I'm watch in the boot the logo but don't load all the sounds and more fast to load

Hacktribe will load `e2sSample.all` from  `KORG/hacktribe/Sample/` directory.   You also need to edit the `e2sSample.all` so the samples start from slot 501.

**Reply by didak00** [reply]
Thank you...but i need the electribe synth right?
1. I take the original synth firmware
2. python e2-header.py SYSTEM.VSB sampler
3. Im put in the KORG/hacktribe/system/SYSTEM.VSB
Yes?
Thank youuu

**Reply by bangcorrupt** [reply]
`/System/` should have a capital 'S', otherwise correct.

**Reply by jdubukurwaq** [reply]
brother i have done all steps, but my electribe keeps saying "no file", the directory is how it should be with all capital letter taken care off, i am completely hopeless what should i do?

**Reply by jdubukurwaq** [reply]
okay so now i have a bricked sampler that doesnt read the ppdated header files, love it :)

**Reply by jdubukurwaq** [reply]
sorry for my frustraions i have fixd it via the debricked page

**Comment by didak00** [comment]
![16746787094328075808981239237293](https://user-images.githubusercontent.com/123538841/214683486-66291af8-e6cb-483f-bd78-fe0e03ec74e9.jpg)

**Comment by didak00** [comment]
Error no file 🙄

**Reply by bangcorrupt** [reply]
Yes.

**Reply by didak00** [reply]
Error. No file
KORG/hacktribe/System/SYSTEM.VSB
😔

**Reply by bangcorrupt** [reply]
Can you email me the file you're trying to install?

**Reply by bangcorrupt** [reply]
That file is correct.  If you have Hacktribe installed and have that file at the correct SD path it should work.  Can you list the files on the SD and post a screenshot?  

I also notice you're using an old version of the script, but in this case it shouldn't make a difference.

**Comment by didak00** [comment]
yes..i cant not understand anything
[cid:ae50460c-d027-4c79-8578-ea0b78451b25]
________________________________
De: bangcorrupt ***@***.***>
Enviado: miércoles, 25 de enero de 2023 21:55
Para: bangcorrupt/hacktribe ***@***.***>
Cc: didak00 ***@***.***>; Comment ***@***.***>
Asunto: Re: [bangcorrupt/hacktribe] How to revert E2 Synth version to the original firmware (Discussion #70)


That file is correct. If you have Hacktribe installed and have that file at the correct SD path it should work. Can you list the files on the SD and post a screenshot?

**Reply by bangcorrupt** [reply]
If you posted an image it didn't work.  Do you have any other files in the `System` directory?  There should be only `SYSTEM.VSB`, (sub-directories are fine),

**Comment by didak00** [comment]
Hi¡¡Nope..maybe the sdcard its not fine...i will try with another one. Thankyou for your help and effort to try turn alive my brick xD.
Could you be send me please a one file system.vsb modified for you and i try with this one? My electribe its grey....if not run..maybe its bricked. I m not another idea.  Cheers
________________________________
De: bangcorrupt ***@***.***>
Enviado: miércoles, 25 de enero de 2023 23:18
Para: bangcorrupt/hacktribe ***@***.***>
Cc: didak00 ***@***.***>; Comment ***@***.***>
Asunto: Re: [bangcorrupt/hacktribe] How to revert E2 Synth version to the original firmware (Discussion #70)


If you posted an image it didn't work. Do you have any other files in the System directory? There should be only SYSTEM.VSB, (sub-directories are fine),

**Comment by didak00** [comment]
![20230126_102524](https://user-images.githubusercontent.com/123538841/214801501-efe6fdc0-12b3-4529-a666-fdfda005e66b.jpg)
Oh sorry im not upload the picture..here you go

**Reply by bangcorrupt** [reply]
Are you running an old version of Hacktribe?  Before 3d31b7c,  Hacktribe would look for files under the `electribe sampler` directory.    Try putting `SYSTEM.VSB` under `KORG/electribe sampler/System/`.  

I tried installing the file you sent me and it reverted to synth firmware correctly.

**Reply by MarcosDelfino** [reply]
Hi there, I cloned and installed Hacktribe on 01/23/23 I don't know how to check for the version. But had to place the firmware under  KORG/electribe sampler/System/ for it to be loaded. Also, the drum samples  and synth sounds are in different locations now. Just letting you now! thank you for all the work you did here!

**Reply by bangcorrupt** [reply]
If you installed for the first time from sampler firmware that is the correct path.  If you upgrade or revert the file will need to be under the `hacktribe/System` directory.

**Comment by didak00** [comment]
oh yeah! yes men!! do it!

100000 milllion thanks

**Reply by herrmautsch** [reply]
How did you do it in the end?

**Comment by vandaltwist** [comment]
hello everyone, I am not the first noob to ask I am sorry I have read several times the creation or modification of script I do not understand anything :( do you have a ready-to-use script please? I would like to pass my hacktribe has electribe 2 blue

**Reply by bangcorrupt** [reply]
Follow the [Hacktribe Tutorial](https://github.com/bangcorrupt/hacktribe-tutorial) to get the patched firmware files.

 Put `synth-revert-SYSTEM.VSB` in the `KORG/hacktribe/System/` directory and rename it to `SYSTEM.VSB`. Then run the update function on the device.

(If you are running a very old version of Hacktribe, you may need to put the update file in the 'KORG/electribe sampler/System/' directory.)

**Reply by bangcorrupt** [reply]
What happens when you click 'Open in a Codespace'?

It should open a new tab, then load the codespace.

**Reply by vandaltwist** [reply]
i undestant nothing as didacticiel hacktribe

**Reply by vandaltwist** [reply]
I don't see the files in the orange code space, only a tutorial that explains things that are too complicated for me :(

**Reply by vandaltwist** [reply]
i need a create this file ? synth-revert-SYSTEM.VSB

**Reply by bangcorrupt** [reply]
Follow the screenshots.  After the codespace opens, wait a while and the script will run automatically.  The files will appear in the file browser on the left.

**Reply by vandaltwist** [reply]
below read me I'm nothing :/

**Reply by vandaltwist** [reply]
![Capture](https://github.com/bangcorrupt/hacktribe/assets/129626139/5df9c2aa-8c28-463a-aa42-5ff274a9cd60)

**Reply by bangcorrupt** [reply]
If you wait a minute the script should run automatically.  If it doesn't, run `./hacktribe-tutorial.sh` in the terminal window at the bottom.


https://github.com/bangcorrupt/hacktribe/assets/58309433/4c563b41-67f2-4274-bbb7-a5196989eacb

**Reply by vandaltwist** [reply]
thank you very much I'm sorry I was so bad, unfortunately there are people like you thank you!!!

**Comment by ghost** [comment]
Hello, 

i've followed everything that you mentionned but my header won't change ... i cannot make kicks anymore only synth. 
i have the grey electribe. 
<img width="756" alt="Capture d’écran 2024-05-01 à 19 54 22" src="https://github.com/bangcorrupt/hacktribe/assets/70519994/c43276d0-b8a0-49c9-926a-e7e8675ec167">
<img width="323" alt="Capture d’écran 2024-05-01 à 19 54 46" src="https://github.com/bangcorrupt/hacktribe/assets/70519994/4dbf120e-119d-4e4c-9e98-29f6700e13ed">

could you help me out ?

**Reply by ghost** [reply]
![IMG_1601](https://github.com/bangcorrupt/hacktribe/assets/70519994/490068f4-cdd8-4267-8a96-77cb8691411b)
![IMG_1602](https://github.com/bangcorrupt/hacktribe/assets/70519994/1bdb0e9e-e670-4f35-8a2d-b6724b848755)

**Reply by bangcorrupt** [reply]
If you use the Hacktribe Tutorial codespace on github you don't need to run any commands.  Just wait a couple of minutes and the files will appear in the browser on the left.  Use `synth-revert-SYSTEM.VSB` to go from Hacktribe to Synth.  Put the file at `KORG/hacktribe/System/SYSTEM.VSB` and `KORG/electribe sampler/System/SYSTEM.VSB`.  Delete any other `*.VSB` files.

**Comment by DOGHATESNOISE** [comment]
Hello I also can't seem to download the file it just opens as a text file when I try to revert my e2 synth version

**Reply by luciferseamus** [reply]
> Hello I also can't seem to download the file it just opens as a text file when I try to revert my e2 synth version
Was the file saved with the .txt extension?

If so remove the .txt

Example:
SYSTEM.vsb.txt

Should be:
SYSTEM.vsb

Edit:  I did a poor job reading.   The file won't save? Like it just opens in a browser window as a text file?

**Reply by bangcorrupt** [reply]
> download the file

Do you mean `synth-revert-SYSTEM.VSB`?

If you right click the file name it should show a context menu allowing you to download.  I just tried it and I had to right click twice to show the menu, the first time it only showed `paste`.

**Reply by DOGHATESNOISE** [reply]
I downloaded it and moved it to the back tribe folder but i can't do a software update to revert it

**Reply by DOGHATESNOISE** [reply]
It's in the back tribe folder on my SD card

**Reply by DOGHATESNOISE** [reply]
Hacktribe*

**Reply by bangcorrupt** [reply]
The file should be in the `System` sub-directory and named `SYSTEM.VSB`.

If you have a very old version of Hacktribe installed, the file needs to be under the `electribe sampler/System` directory.

Put the file at both paths to make sure.

`KORG/hacktribe/System/SYSTEM.VSB`

`KORG/electribe sampler/System/SYSTEM.VSB`

https://github.com/bangcorrupt/hacktribe/wiki/How-To#factory-synth

**Comment by DOGHATESNOISE** [comment]
Thank you! I was able to figure it out about an hour ago and now it's back
to normal. Missed all the stock oscillators. Hacking it was a fun adventure
tho I'm glad I know I can reach it if I need to

**Comment by fedealjeno** [comment]
hi i putted the original synth firmwarein the system folder hit update to go back without formatting it from the korg and now it says update and no file even with the right file in thre right folder. do i have to format the sd throw onother e2?v or i ruined forever the machine? please help im a noob at coding Thanks

**Reply by vanana-old-acc** [reply]
So you cannot start your e2 anymore?

**Reply by fedealjeno** [reply]
![image](https://github.com/user-attachments/assets/911d75e6-1dee-471c-80de-89c9cf3fc9d6)
![image](https://github.com/user-attachments/assets/4c63af80-19d9-434b-8b62-74a7a854e871)
![image](https://github.com/user-attachments/assets/9c178c79-d293-42cb-9641-ac114f4a5c82)

**Reply by fedealjeno** [reply]
these are the only things i see i tried everything but format the sd card with another e2 and not from the pc

**Reply by vanana-old-acc** [reply]
Unfortunately, you need to reflash your machine: it broke. You need to attach the clip to the flash chip on the motherboard, and backup old flash, and then write a new flash dump to it. You will lose all your patterns, but if you send me the backup flash, I can recover them for you https://www.amazon.nl/-/en/DollaTek-Soic8-eeprom-ch341a-programmer/dp/B08HQB1VL8

If you need more help I can assist

**Reply by fedealjeno** [reply]
> Unfortunately, you need to reflash your machine: it broke. You need to attach the clip to the flash chip on the motherboard, and backup old flash, and then write a new flash dump to it. You will lose all your patterns, but if you send me the backup flash, I can recover them for you https://www.amazon.nl/-/en/DollaTek-Soic8-eeprom-ch341a-programmer/dp/B08HQB1VL8
> 
> 
> 
> If you need more help I can assist

the item is not available in my country is this good?

https://a.aliexpress.com/_ExDpxge

**Reply by fedealjeno** [reply]
> Unfortunately, you need to reflash your machine: it broke. You need to attach the clip to the flash chip on the motherboard, and backup old flash, and then write a new flash dump to it. You will lose all your patterns, but if you send me the backup flash, I can recover them for you https://www.amazon.nl/-/en/DollaTek-Soic8-eeprom-ch341a-programmer/dp/B08HQB1VL8
> 
> 
> 
> If you need more help I can assist

and ye i need assistance during the process i've never done something like this thank you

**Reply by ghost** [reply]
> you need to reflash your machine: it broke.

You shouldn't need to do that.  Get `synth-revert-SYSTEM.VSB` from the [Hacktribe Tutorial](https://github.com/bangcorrupt/hacktribe-tutorial), put it at `KORG/electribe sampler/System/SYSTEM.VSB`.

**Reply by fedealjeno** [reply]
already did that

**Reply by ghost** [reply]
It looks like you have the stock sampler firmware installed.  Do you definitely have it under `electribe sampler` directory, not just `electribe` or `hacktribe`?  Try putting it at all 3 paths.

**Reply by fedealjeno** [reply]
yes i already did it ,right files right folders always same error

**Reply by fedealjeno** [reply]
thanks to all of you guys for helping me tho hope we can save this E2!

**Reply by ghost** [reply]
Do you have any other files in the `System` directories?  It will only try the first file it finds (sub-directories are ok).

**Comment by fedealjeno** [comment]
thanks

**Comment by fedealjeno** [comment]
i will try with my friend on tuesday i will probably buy the stuff sorry i
really know nothing about coding and hardware i just play music
thank you so much for helping me

**Comment by jdubukurwaq** [comment]
Bro you have to adjust the code in python,  you have to adjust the header
and then the synth thinks it's a synth file

---------- Forwarded message ---------
Van: fedealjeno ***@***.***>
Date: ma 6 jul. 2026 12:09
Subject: Re: [bangcorrupt/hacktribe] How to revert E2 Synth version to the
original firmware (Discussion #70)
To: bangcorrupt/hacktribe ***@***.***>
Cc: jdubukurwaq ***@***.***>, Comment <
***@***.***>


already did that

—
Reply to this email directly, view it on GitHub
<https://github.com/bangcorrupt/hacktribe/discussions/70?email_source=notifications&email_token=BP7PR5RFUHIUJWLAANNCT635DN3HHA5CNFSNUABIM5UWIORPF5TWS5BNNB2WEL2ENFZWG5LTONUW63SDN5WW2ZLOOQXTCNZVGQ3DANZWUZZGKYLTN5XKOY3PNVWWK3TUUVSXMZLOOSWGM33PORSXEX3DNRUWG2Y#discussioncomment-17546076>,
or unsubscribe
<https://github.com/notifications/unsubscribe-auth/BP7PR5TIJ2U5UMKGHSFJGCL5DN3HHAVCNFSNUABHKJSXA33TNF2G64TZHM2DCNBXGA4TINRWHNCGS43DOVZXG2LPNY5TGNZZGYYDINVBOYBA>
.
Triage notifications, keep track of coding agent tasks and review pull
requests on the go with GitHub Mobile for iOS
<https://github.com/notifications/mobile/ios/BP7PR5X4MVJB4KPV35AXGNL5DN3HHA5CNFSNUABIM5UWIORPF5TWS5BNNB2WEL2ENFZWG5LTONUW63SDN5WW2ZLOOQXTCNZVGQ3DANZWUZZGKYLTN5XKOY3PNVWWK3TUUVSXMZLOOSVGM33PORSXEX3JN5ZQ>
and Android
<https://github.com/notifications/mobile/android/BP7PR5WOJUM3C76J43JQ77D5DN3HHA5CNFSNUABIM5UWIORPF5TWS5BNNB2WEL2ENFZWG5LTONUW63SDN5WW2ZLOOQXTCNZVGQ3DANZWUZZGKYLTN5XKOY3PNVWWK3TUUVSXMZLOOSXGM33PORSXEX3BNZSHE33JMQ>.
Download it today!
You are receiving this because you commented.Message ID:
***@***.***>

**Reply by fedealjeno** [reply]
the revertSystem file isn't the system file already modified?

**Comment by wagnertschis3-cloud** [comment]
Hallo Leute kann mir wer helfen die Electribe 2 rot Sampler von Hacktribe wieder auf Werkseinstellungen oder original Software zurück zu setzen bitte Danke Leute im Vorhinein
